lib/warnings.js

```diff
@@ -27,4 +27,6 @@ warning.create('FastifyDeprecation', 'FSTDEP013', 'Direct return of "trailers" f
 
 warning.create('FastifyDeprecation', 'FSTDEP014', 'You are trying to set/access the default route. This property is deprecated. Please, use setNotFoundHandler if you want to custom a 404 handler or the wildcard (*) to match all routes.')
 
+warning.create('FastifyDeprecation', 'FSTDEP015', 'The %s schema for %s: %s is undefined. This may indicate the schema is not well specified.')
```

Comment:
Shall we indicate that this behavior will change in `v5`?

Reply:
Will we keep throwing the error (#4620) on `v5`, @Eomm? If yes, then we can modify the warning exception message to something like this:

> The params schema for GET: /:id is missed. This may indicate the schema is not well specified. Ir will cause a runtime error building the validation schema in `fastify@5`.'

WDYT?

Reply:
Okay, I will keep the current message if this is not an issue. For instance:

> The params schema for GET: /:id is missed. This may indicate the schema is not well specified. It will cause a runtime error building the validation schema in fastify@5.'